test-suite/src/index/showindexes.rs

```diff
@@ -36,9 +39,25 @@ CREATE TABLE Test (
 
     test!(
         Ok(Payload::ShowIndexes(vec![
-            ("idx_id".to_string(), SchemaIndexOrd::Both),
-            ("idx_name".to_string(), SchemaIndexOrd::Both),
-            ("idx_id2".to_string(), SchemaIndexOrd::Both)
+            (SchemaIndex {
+                name: "idx_id".to_string(),
+                order: SchemaIndexOrd::Both,
+                expr: Expr::Identifier("id".to_string())
+            }),
+            (SchemaIndex {
+                name: "idx_name".to_string(),
+                order: SchemaIndexOrd::Both,
+                expr: Expr::Identifier("name".to_string())
+            }),
+            (SchemaIndex {
+                name: "idx_id2".to_string(),
+                order: SchemaIndexOrd::Both,
+                expr: Expr::BinaryOp {
+                    left: Box::new(Expr::Identifier("id".to_string())),
+                    op: BinaryOperator::Plus,
+                    right: Box::new(Expr::Identifier("num".to_string()))
+                }
+            })
```

Comment:
`()` bracket looks unnecessary.
```rust
SchemaIndex {
     name: "idx_id".to_string(),
     order: SchemaIndexOrd::Both,
     expr: Expr::Identifier("id".to_string())
},
```
would be enough without `(SchemaIndex { .. })`